godot-core/src/builtin/string.rs

```diff
@@ -147,6 +149,24 @@ impl fmt::Debug for GodotString {
     }
 }
 
+impl ToVariant for &str {
+    fn to_variant(&self) -> Variant {
+        crate::builtin::GodotString::from(*self).to_variant()
```

Comment:
`GodotString` is in scope here, no need to qualify the module path.
Same below.

Reply:
oh true lol, i moved it from another place where i needed to qualify it